ivy/functional/frontends/tensorflow/general_functions.py

```diff
@@ -80,6 +81,23 @@ def fill(dims, value, name=None):
     return ivy.full(dims, value)
 
 
+@to_ivy_arrays_and_back
+def foldl(fn, elems, initializer=None):
```

Comment:
Please implement the exact same signature of [foldl](https://www.tensorflow.org/api_docs/python/tf/compat/v1/foldl)